### ISSUE TITLE
File tree on "Night" theme cascades diagonally; resolved with `display: grid` on `.file-node-children`

### ISSUE THREAD

**Issue by jkingsman** [typora/typora-issues#5253]
- [X] Searched existing issues to avoid creating duplicates.

- [X] Confirmed that it can be reproduced in built-in themes without customized css.

   If it only exists in 3rd party themes or css, you can still report it, but please attach the theme target or the css file. We may not "fix" it, if it is caused by 3rd party themes or css styles that we cannot support.

- [X] Searched http://support.typora.io/

----

**Describe the bug**
The "Night" builtin theme creates a list of cascading files diagonally rather than straight down.

**To Reproduce**
1. Select the "Night" theme
2. Enable the file tree sidebar

**Expected behavior**
A typical vertical list.

**Screenshots / Screencasts**
![image](https://user-images.githubusercontent.com/2365891/168853625-8e3e8c5d-18cc-4733-a118-93037640f29c.png)

**Sample Markdown File**
N/A

**Desktop (please complete the following information):**
 - MacOS Cataline 10.15.7

**Typora Version**
1.2.3 (6303) although problem has persisted for some time.

**Additional context**
Setting `display: grid` on `.file-node-children` fixes the issue in the debugger.

I'm comfortable using and hot-modifying the page with the inspector if there's something helpful I can do re: testing or more information.

**Comment by abnerlee** [comment]
I do not have `MacOS Catalina 10.15.7`, on macOS 12, night theme works fine.

But I will add `.file-node-expanded>.file-node-children {display: grid;}` for Night theme in next veriosn

**Comment by jkingsman** [comment]
Aha, I was bothered by the solution and found the actual issue. 

```
.html-for-mac .file-node-open-state, .html-for-mac .file-node-icon {
    line-height: 26px;
}
```

is causing the issue. The icon is dropping into the next line's space and forcing it to offset before beginning the line. Reducing to `25px` fixes the issue but that may just be a vagary of my display setup or monitor or something. 

The grid fix also fixes it since it just pushes the start of the next row down a bit (as in a grid display) instead of trying to smush them together and offsetting the start until it has sufficient clearance (I think -- frontend is not my area of expertise). 

So, grid fix is fine (and possibly good to enforce globally on the file tree so that any variation in line height doesn't cause that cascade) even though the root cause is a line height over-extension. In fact, I'd say grid is ideal since it addresses any permutation of the problem, but if you want a more targeted fix, there ya go.

**Comment by AugustM** [comment]
I have the same issue with the Night theme on Mojave.

Abner says this be fixed in the next Typora release with the "grid fix". 

What customization to the Night theme I can apply in `night.user.css` to fix it now?

**Comment by jkingsman** [comment]
```
.file-node-children {
    display: grid;
}
```

should do the trick for you.

**Comment by AugustM** [comment]
Thanks. The instructions in `https://support.typora.io/Add-Custom-CSS/` suggest that adding a file named `night.user.css` in the theme folder containing simply those 3 lines should be enough. I’ll try it.

**Comment by abnerlee** [comment]
fixed in 1.3.6
